src/main/proto/core/utwin/v1/utwin.proto

```diff
@@ -38,17 +38,12 @@ service uTwin {
   option (version_minor) = 0;
```

Comment:
should the major version change as GetLastEvent has been removed ?

Reply:
No cuz we never did anything yet with 1.4 uTwin, the plan is to use 1.5 uTwin definitions